cli/ballerina-cli/src/main/resources/cli-help/ballerina-persist.help

```diff
@@ -0,0 +1,53 @@
+NAME
+       bal persist - Manage data persistence
+
+SYNOPSIS
+       bal persist <command> [<-h> | <--help>]
+
+DESCRIPTION
+       Manage the data persistence in a Ballerina package.
+
+       Initializing the package for persistence is the first step toward achieving
+       data persistence in a Ballerina package. This step creates the required
```

Comment:
```suggestion
       data persistence in a Ballerina package. This step creates the required
```
Shall we make the steps numbered bullet points?

Reply:
As this is a `help` description in the shell, better if we go with paragraph form